packages/search-ui-app-search-connector/src/AppSearchAPIConnector.js

```diff
@@ -3,6 +3,7 @@ import { version } from "../package.json";
 
 import { adaptResponse } from "./responseAdapter";
 import { adaptRequest } from "./requestAdapters";
+import buildResponseAdapterOptionsFromConfig from "./buildResponseAdapterOptionsFromConfig";
```

Comment:
I think it's going to take me a bit longer to fully read through/understand this new helper, so I'm gonna submit my comments for now and do another pass with the remaining files in a bit here, if that's ok!

My one comment here (feel free to disregard me also if you feel otherwise - I don't feel super strongly about this) is that `buildResponseAdapterOptionsFromConfig` is a bit verbose (although of course very explicit, which is great) - do you think we'd lose anything if we simplified it to, e.g. `buildResponseAdapterOptions` or maybe `buildResponseOptions`?

Reply:
I appreciate your thoughtfulness. You're giving this way more thought than I did at the time of writing. I frankly just wanted to separate out this code so I could test that logic independently. I will rename it to `buildResponseAdapterOptions`.